Source/JavaScriptCore/jsc.cpp

```diff
@@ -4422,8 +4422,8 @@ int jscmain(int argc, char** argv)
 
 #if OS(WINDOWS)
     // Needed for complex.yaml tests.
-    if (char* tz = getenv("TZ"))
-        setTimeZoneOverride(StringView::fromLatin1(tz));
+    if (auto tz = unsafeNullTerminated(getenv("TZ")))
+        setTimeZoneOverride(StringView(tz));
```

Comment:
I am not thrilled by the change from explicitly Latin-1 to something implicit. Am I missing something?